assets/js/src/core/modules/redirects/index.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * This source file is available under the terms of the
+ * Pimcore Open Core License (POCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (https://www.pimcore.com)
+ *  @license    Pimcore Open Core License (POCL)
+ */
+
+import { type WidgetRegistry } from '@Pimcore/modules/widget-manager/services/widget-registry'
+import { container } from '@Pimcore/app/depency-injection'
+import { serviceIds } from '@Pimcore/app/config/services/service-ids'
+import { moduleSystem } from '@Pimcore/app/module-system/module-system'
+import { type MainNavRegistry } from '../app/base-layout/main-nav/services/main-nav-registry'
+// import { NavPermission } from '../perspectives/enums/nav-permission'
+// import { UserPermission } from '../auth/enums/user-permission'
```

Comment:
Followup to: https://github.com/pimcore/studio-ui-bundle/pull/1777/files#r2247367995

With the working permission in place we can enable these checks

```suggestion
import { NavPermission } from '../perspectives/enums/nav-permission'
import { UserPermission } from '../auth/enums/user-permission'
```